lib/src/test/java/io/ably/lib/test/realtime/RealtimePresenceTest.java

```diff
@@ -2062,7 +2067,8 @@ public void onPresenceMessage(PresenceMessage message) {
 				}
 			});
 
-			PresenceMessage[] testData = new PresenceMessage[] {
+			/* Test message newness criteria as described in RTP2b */
+			final PresenceMessage[] testData = new PresenceMessage[] {
```

Comment:
Added timestamps to the messages containing correct conenctionId to check newness comparison priority